trans/templates/home.html

```diff
@@ -156,53 +156,70 @@ <h4> You are translating into <b>{{ user.language.name }} ({{ user.language.code
           <h3 style="color:green">
             All translations for {{ contest.title }} have been finalized for team {{ user.username }}. If this is an error, please contact a member of the HTC/ITC.
           </h3>
-          <br>
+          {% with extra_country_1_code=contest.user_contest.extra_country_1_code %}
+          {% with extra_country_1_count=contest.user_contest.extra_country_1_count %}
+          {% with extra_country_2_code=contest.user_contest.extra_country_2_code %}
+          {% with extra_country_2_count=contest.user_contest.extra_country_2_count %}
+          {% if extra_country_1_count == 0 and extra_country_2_count == 0 %}
+            <h4>Requested additional copies: None</h4>
+          {% else %}
+            <h4>Requested additional copies:</h4>
+            <ul>
+              {% if extra_country_1_count > 0 and extra_country_1_code %}
+                <li>{{ extra_country_1_code }}: {{ extra_country_1_count }}

```

Comment:
Should we add more details (language & language.code) for the finalised extra copies..?

Reply:
Done. A bit hacky though. We should probably make a one-to-one relationship from country to User.